packages/core/upload/tests/admin/folder.test.e2e.js

```diff
@@ -326,4 +326,192 @@ describe('Folder', () => {
       data.folders.push(folder1, folder1b);
     });
   });
+
+  describe('update', () => {
+    test('rename a folder', async () => {
+      const folder = await createFolder('folder-name', null);
+
+      const res = await rq({
+        method: 'PUT',
+        url: `/upload/folders/${folder.id}`,
```

Comment:
For assets we are using a GET param: `?id=${file.id}`. I'd like to keep this behaviour in sync between both content-types. What do you think - should we update files or folders?

Reply:
Files should be updated then, would it be ok for you to do it the front?
Folders follow our new API standard

Reply:
I'm ok with that.